app/models/organization.rb

```diff
@@ -1,3 +1,6 @@
 class Organization < ApplicationRecord
   has_and_belongs_to_many :users
+  has_many :attachments, as: :attachable, dependent: :destroy
+  accepts_nested_attributes_for :attachments
+  attr_accessor :attachment
```

Comment:
Does not work without the `attr_accessor :attachment` line.
Error: unknown attribute 'attachment' for Organization.